python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1746,6 +1766,32 @@ def findArtifacts(self, templateCoadd, tempExpRefList, imageScalerList):
                              'EDGE': edge})
         return altMasks
 
+    def _prefilterArtifacts(self, spanSetList, exp):
```

Comment:
Suggest dropping the leading underscore. I don't think the method is as private as this indicates.